components/alert/alert.css

```diff
@@ -0,0 +1,44 @@
+.alert-container {
+  max-width: 50rem;
+  display: inline-block;
```

Comment:
using flex would be a better option.

Reply:
yeah I could use but it , but I tried to make separate class, and flex would change my different element also,  that's why 
 I used this.
Thanks for reviewing my PR